docs/docsite/rst/user_guide/vault.rst

```diff
@@ -424,6 +429,58 @@ When using :ref:`ansible-vault` commands that encrypt content (:ref:`ansible-vau
 only one vault-id can be used.
 
 
+.. _vault_securing_editor:
+Securing your editor
+````````````````````
+
+As noted above, the vault system relies on your configured ``$EDITOR`` which can be a source of disclosures, most editors have conviniences to prevent loss of data, but this normaly relies
+on extra plain text files that can have a clear text copy of your secrets. To prevent this you should consult your editor's manual to ensure you configure it to avoid disclosing secure data.
+
+
+vim
+^^^
+
+These are some well known options you can set in command mode to avoid cases of disclosure, be aware that there are probably more settings you'll need to modify to ensure security, specially when using plugins.
```

Comment:
```suggestion
These are some well known options you can set in command mode to avoid cases of disclosure. Be aware that there are probably more settings you'll need to modify to ensure security, especially when using plugins.
```